snippet_ranger/dependent_reps.py

```diff
@@ -0,0 +1,157 @@
+import json
+import logging
+import os
+
+import pandas as pd
+
+from ast2vec.pickleable_logger import PickleableLogger
+from modelforge.progress_bar import progress_bar
+
+dependencies_filename = 'repository_dependencies-1.0.0-2017-06-15.csv'
+repos_filename = 'repositories-1.0.0-2017-06-15.csv'
+projects_filename = 'projects-1.0.0-2017-06-15.csv'
+
+
+class LibIO(PickleableLogger):
```

Comment:
Come on. Class names shall be as much verbose as possible. `LibrariesIOFetcher` or whatever similar